### PR TITLE
TASK-66402: add iframely standalone support to acceptance

### PR DESCRIPTION
**Pull request by bourasom**

### CODE REVIEW

File: _functions.sh [hunk 1]
```diff
@@ -2246,7 +2269,10 @@ do_undeploy() {
       # Close firewall port for LDAPS
       do_ufw_close_port ${DEPLOYMENT_LDAP_PORT} "Ldap Port" ${ADT_DEV_MODE}
     fi
-    
+    if [ "${DEPLOYMENT_IFRAMELY_ENABLED}" == "true" ]; then
+      # Close firewall port for IFRAMELY
+      do_ufw_close_port ${DEPLOYMENT_IFRAMELY_PORT} "IFRAMELY Port" ${ADT_DEV_MODE}
```
Comment (hbenali) [inline _functions.sh:2274]: Why do we need this?

File: _functions.sh [hunk 2]
```diff
@@ -2028,6 +2046,9 @@ do_start() {
   if [ ! -z "${DEPLOYMENT_LDAP_LINK}" ]; then
     echo_info "LDAP URL  : ${DEPLOYMENT_LDAP_LINK}"
   fi
+  if [ ! -z "${DEPLOYMENT_IFRAMELY_LINK}" ]; then
+    echo_info "IFRAMELY URL  : ${DEPLOYMENT_IFRAMELY_LINK}"
+  fi
```
Comment (hbenali) [inline _functions.sh:2051]: Do we need really this ? IMO it has be protected (eg hosting 403 error due to cors policies)

File: _functions.sh (hunk 1 above)
Comment (bourasom) [reply _functions.sh:2274]: Not really, it was taken from the ldap conf

Comment (bourasom) [inline _functions.sh:2274]:
```suggestion

```

File: _functions.sh (hunk 2 above)
Comment (bourasom) [reply _functions.sh:2051]: will not really foccus on that, no critical data on the container

File: _functions.sh [hunk 3]
```diff
@@ -2246,7 +2269,9 @@ do_undeploy() {
       # Close firewall port for LDAPS
       do_ufw_close_port ${DEPLOYMENT_LDAP_PORT} "Ldap Port" ${ADT_DEV_MODE}
     fi
-    
+    if [ "${DEPLOYMENT_IFRAMELY_ENABLED}" == "true" ]; then
+      # Close firewall port for IFRAMELY
+    fi
```
Comment (hbenali) [inline _functions.sh:2274]:
```suggestion
```

File: _functions.sh [hunk 4]
```diff
@@ -1728,6 +1741,10 @@ do_deploy() {
     DEPLOYMENT_LDAP_LINK="ldap://${DEPLOYMENT_EXT_HOST}:${DEPLOYMENT_LDAP_PORT}"   
   fi
 
+  if [ "${DEPLOYMENT_IFRAMELY_ENABLED}" == "true" ]; then
+    DEPLOYMENT_IFRAMELY_LINK="http://${DEPLOYMENT_EXT_HOST}:${DEPLOYMENT_IFRAMELY_PORT}"
```
Comment (hbenali) [inline _functions.sh:1745]:
```suggestion
    DEPLOYMENT_IFRAMELY_LINK="${DEPLOYMENT_URL}/oembed"
```

**Review by hbenali — COMMENTED** [review]
Due to firewall restriction we need to use frontend link instead

**Review by hbenali — APPROVED** [review]

**Pull request merged**